src/Apps/Collect/CollectApp.tsx

```diff
@@ -96,6 +96,8 @@ export const CollectAppFragmentContainer = createFragmentContainer(
         }
         sort: { type: "String", defaultValue: "-partner_updated_at" }
         price_range: { type: "String" }
+        height_range: { type: "String" }
```

Comment:
#minor since these eventually become `height` and `width` when passed into the GraphQL resolver (in https://github.com/artsy/reaction/pull/1948/files#diff-9f168adbb6f7152f905d1a0f67188475R106), I would maybe just call them that throughout (avoiding `_range` entirely, and avoiding needing to remember to rename them before using them in the GraphQL resolver)

Reply:
hmm yeah I hear that. I kinda like specifying `_range` because it makes the reference a bit clearer everywhere it appears and it more accurately reflects the actual data (if I'm understanding the query correctly—it's filtering the artworks by height but using a string representing the height range, right? E.g. `5000-10000`). is that more or less correct?

Reply:
My vote would be to stay consistent, even if that means sticking with an attribute name that doesn't perfectly represent  the underlying value.

If we hadn't already decided on `height` and `width` filter attributes in Metaphysics, we could implement `heigh_range` and `width_range` attribute names across the board. But for now, I think it's more important for the names to stay consistent as much as possible than to improve upon the attribute names in this particular part of the system.

It looks like we might already be forgetting the height: `height_range / width: width_range` rename elsewhere in this graphql block.

@mdole Does that sounds good to you?

Reply:
point taken! that does make sense. will update throughout